segmentstore/server/src/main/java/io/pravega/segmentstore/server/store/ServiceConfig.java

```diff
@@ -42,7 +42,9 @@ public class ServiceConfig {
     public static final Property<Integer> CONTAINER_COUNT = Property.named("container.count", null, "containerCount");
     public static final Property<Integer> PARALLEL_CONTAINER_STARTS = Property.named("container.parallelStarts", 2);
     public static final Property<Integer> THREAD_POOL_SIZE = Property.named("threadPool.core.size", 30, "threadPoolSize");
-    public static final Property<Integer> STORAGE_THREAD_POOL_SIZE = Property.named("threadPool.storage.size", 200, "storageThreadPoolSize");
+    public static final Property<Integer> STORAGE_THREAD_POOL_SIZE = Property.named("threadPool.storage.size", 50, "storageThreadPoolSize");
```

Comment:
You are changing default thread pool sizes without any evidence on how this is going to impact the performance of moving data to LTS.

Reply:
Raised this PR for review, meanwhile I will be running the performance.